sjb/config/test_cases/test_branch_ansible_service_broker.yml

```diff
@@ -0,0 +1,93 @@
+        source hack/lib/init.sh
+        export ORIGIN_BUILT_VERSION=$(os::build::rpm::format_nvra)
+        echo $ORIGIN_BUILT_VERSION > "${jobs_repo}/ORIGIN_BUILT_VERSION"
+    - type: "script"
+      title: "build Origin"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64 hack/env make build
+    - type: "script"
+      title: "build Service Catalog binary"
+      repository: "service-catalog"
+      timeout: 1800
+      script: |-
+        hack/env make build
+    - type: "script"
+      title: "build Service Catalog image"
+      repository: "service-catalog"
+      timeout: 1800
+      script: |-
+        hack/env hack/build-source-image.sh
+        hack/env make build-images
+    - type: "script"
+      title: "build origin images"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/build-local-images.py
+    - type: "script"
+      title: "start cluster with Service Catalog"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        if [[ "${PULL_BASE_REF:-master}" == "release-3.9" ]]; then
+          ./_output/local/bin/linux/amd64/oc cluster up --loglevel=5 --version=latest --service-catalog
+        else
+          ./_output/local/bin/linux/amd64/oc cluster up --loglevel=5 --tag=latest --enable=service-catalog
+        fi
+        ./_output/local/bin/linux/amd64/oc login -u system:admin
+        ./_output/local/bin/linux/amd64/oc describe po --all-namespaces
+    - type: "script"
+      title: "Install ASB to cluster"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        ./_output/local/bin/linux/amd64/oc create project ansible-service-broker
+        ./_output/local/bin/linux/amd64/oc process 'https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/deploy-ansible-service-broker.template.yaml' -p BROKER_CA_CERT=$(oc get secret -n kube-service-catalog -o go-template=\'{{ range .items }}{{ if eq .type \"kubernetes.io/service-account-token\" }}{{ index .data \"service-ca.crt\" }}{{end}}{{\"\\n\"}}{{end}}\' | tail -n 1) | oc create -n ansible-service-broker -f -'
+    - type: "script"
+      title: "Relist Catalog to pickup new clusterserviceclasses"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        # relistRequets is intially zero, and to trigger relist, need to increment. Patch it to force 1.
+        ./_output/local/bin/linux/amd64/oc patch clusterservicebroker ansible-service-broker -p '{"spec":{"relistRequests":1}}'
+    - type: "script"
+      title: "Install ASB CI tool"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        go get github.com/rthallisey/service-broker-ci/cmd/ci
+    - type: "script"
+      title: "Run ASB ci"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        cd $GOPATH/src/openshift/ansible-service-broker && ci
```

Comment:
I think if you change the repo line on this one and above (probably to "ansible-service-broker"), you don't need to worry about using cd. ./ci instead of just ci though.